source/ota.c

```diff
@@ -1903,54 +1902,52 @@ static OtaErr_t validateUpdateVersion( const OtaFileContext_t * pFileContext )
     return err;
 }
 
-/* If there is an error is parsing the json check if it can be handled by external callback. */
-
-static OtaJobParseErr_t parseJobDocFromCustomCallback( const char * pJson,
-                                                       uint32_t messageLength )
+/* If there is an error in parsing the json, check if it can be handled by external callback. */
+static OtaJobParseErr_t handleCustomJob( const char * pJson,
+                                         uint32_t messageLength )
```

Comment:
messageLength should be size_t

Reply:
Same as [this](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/164#discussion_r577883060) reply.